File: Operators-And-Expressions/OddOrEvenIntegers/OddOrEven.cs
// Problem 1. Odd or Even Integers

// Write an expression that checks if given integer is odd or even.

using System;

class OddOrEven
{
    static void Main()
    {
        Console.WriteLine("Insert number: ");
        int n = int.Parse(Console.ReadLine());
        Console.WriteLine("Number " + n + " is " + ((n % 2 == 0) ? "even" : "odd") + ".");
    }
}

File: JavaScript Fundamentals Homeworks/Operators and Expressions/odd-or-even.js
//Problem 1. Odd or Even
//Write an expression that checks if given integer is odd or even.

var even = 4;
var odd = 3;

console.log(isEven(even));
console.log(isEven(odd));

function isEven(number){
    return number % 2 === 0;
}